src/main/java/dev/zenith/trader/EnchantmentUtil.java

```diff
@@ -0,0 +1,86 @@
+package dev.zenith.trader;
+
+import com.google.gson.Gson;
+import com.google.gson.reflect.TypeToken;
+import com.zenith.mc.enchantment.EnchantmentData;
+import com.zenith.mc.enchantment.EnchantmentRegistry;
+import com.zenith.mc.item.ItemRegistry;
+import it.unimi.dsi.fastutil.ints.Int2IntArrayMap;
+import it.unimi.dsi.fastutil.ints.Int2IntMap;
+import org.geysermc.mcprotocollib.protocol.data.game.item.ItemStack;
+import org.geysermc.mcprotocollib.protocol.data.game.item.component.DataComponentTypes;
+import org.geysermc.mcprotocollib.protocol.data.game.item.component.DataComponents;
+import org.geysermc.mcprotocollib.protocol.data.game.item.component.ItemEnchantments;
+
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import java.util.stream.Collectors;
+
+public class EnchantmentUtil {
+
+    public static final String ENCHANTMENT_LEVEL_MAX = "{\"aqua_affinity\":1,\"flame\":1,\"silk_touch\":1,\"mending\":1,\"infinity\":1,\"channeling\":1,\"binding_curse\":1,\"vanishing_curse\":1,\"multishot\":1,\"fire_aspect\":2,\"punch\":2,\"knockback\":2,\"frost_walker\":2,\"unbreaking\":3,\"sweeping_edge\":3,\"luck_of_the_sea\":3,\"lure\":3,\"wind_burst\":3,\"looting\":3,\"depth_strider\":3,\"riptide\":3,\"thorns\":3,\"quick_charge\":3,\"fortune\":3,\"swift_sneak\":3,\"soul_speed\":3,\"respiration\":3,\"loyalty\":3,\"protection\":4,\"projectile_protection\":4,\"blast_protection\":4,\"breach\":4,\"piercing\":4,\"feather_falling\":4,\"fire_protection\":4,\"bane_of_arthropods\":5,\"efficiency\":5,\"sharpness\":5,\"density\":5,\"smite\":5,\"impaling\":5,\"power\":5}";
```

Comment:
if we want this data it needs changes in the main zenith codebase, not a big json blob

the data can be extracted from the network registry into the `EnchatmentRegistry`'s `EnchantmentData` objects

not your fault and i'd prefer doing this myself, in the next zenith update